### ISSUE TITLE
Add Inshorts Clone

### ISSUE THREAD

**Issue by bosedipayan** [Rayman-Sodhi/Clone-IT#67]
Project title
A full clone website of inshorts news app.

Project Description
Hi @Rayman-Sodhi , I would love to create a full clone website of inshorts news app.
Thank You

Stack:
React Js
Material UI
Newsapi

**Comment by bosedipayan** [comment]
As a GSsOC participant, I would love to work on it.

**Comment by Rayman-Sodhi** [comment]
Hi @bosedipayan    you can work on this issue ur project should have a working link and you should add that in the main website too once done with creating it .

Deadline -  5-6 days

**Comment by gurjeetsinghvirdee** [comment]
Hey, @bosedipayan are you still working on this issue? and you crossed your deadline, reply in 24hr either I will close this issue

**Comment by bosedipayan** [comment]
Hey, @gurjeetsinghvirdee the issue is already completed, I'm getting an error in the deployment link, the only reason I'm not giving PR. Could you please guide me out ?

**Comment by gurjeetsinghvirdee** [comment]
> Hey, @gurjeetsinghvirdee the issue is already completed, I'm getting an error in the deployment link, the only reason I'm not giving PR. Could you please guide me out ? 

Deploy with github or netlify

**Comment by bosedipayan** [comment]
Yes, I deployed but still the data is creating issue on chrome. Okay, I'm creating PR soon.
